src/test/java/com/newworld/saegil/authentication/service/AuthenticationServiceTest.java

```diff
@@ -188,21 +176,25 @@ class Context_with_invalid_token {
                 // given
                 final User user = new User("권예진", "http://example.com/profile.jpg", "1234567", OAuth2Type.KAKAO);
                 userRepository.save(user);
-                final Token token = tokenProcessor.generateToken(LocalDateTime.now(), new PrivateClaims(user.getId()).toMap());
+                final Token token = tokenProcessor.generateToken(LocalDateTime.now(),
+                        new PrivateClaims(user.getId()).toMap());
                 blacklistTokenRepository.save(new BlacklistToken(user.getId(), TokenType.ACCESS, token.accessToken()));
-                blacklistTokenRepository.save(new BlacklistToken(user.getId(), TokenType.REFRESH, token.refreshToken()));
+                blacklistTokenRepository.save(
+                        new BlacklistToken(user.getId(), TokenType.REFRESH, token.refreshToken()));
 
                 entityManager.flush();
                 entityManager.clear();
 
                 // when & then
                 SoftAssertions.assertSoftly(softAssertions -> {
-                    softAssertions.assertThatThrownBy(() -> authenticationService.getValidPrivateClaims(TokenType.ACCESS, token.accessToken()))
-                                  .isInstanceOf(InvalidTokenException.class)
-                                  .hasMessage("사용할 수 없는 토큰입니다.");
-                    softAssertions.assertThatThrownBy(() -> authenticationService.getValidPrivateClaims(TokenType.REFRESH, token.refreshToken()))
-                                  .isInstanceOf(InvalidTokenException.class)
-                                  .hasMessage("사용할 수 없는 토큰입니다.");
+                    softAssertions.assertThatThrownBy(
+                                    () -> authenticationService.getValidPrivateClaims(TokenType.ACCESS, token.accessToken()))
+                            .isInstanceOf(InvalidTokenException.class)
+                            .hasMessage("사용할 수 없는 토큰입니다.");
```

Comment:
여기도 마찬가지입니다. assertThatThrownBy, isInstanceOf, hasMessage 가 중요하기 때문에 개행을 해서 중간에 흐름을 끊기 보다는 isInstanceOf가 assertThatThrownBy 바로 다음 줄에 위치하는게 더 가독성이 좋다고 판단했습니다. 이렇게 해도 assertThatThrownBy 절이 지나치게 길어지지 않으니까요.

그래도 개행을 해야 한다면 아래처럼 assertThatThrownBy절 괄호의 시작과 끝을 명확히 해주는게 좋다고 생각합니다. 지금 코드로 읽다보면 괄호를 하나하나 세지 않는 이상 .isInstanceOf가 .getValidPrivateClaims() 뒤에 걸리는 것인지 assertThatThrownBy() 뒤에 걸리는 것인지 명확하지 않다고 생각해요.
```suggestion
                    softAssertions.assertThatThrownBy(
                                          () -> authenticationService.getValidPrivateClaims(TokenType.ACCESS, token.accessToken())
                                  ).isInstanceOf(InvalidTokenException.class)
                                  .hasMessage("사용할 수 없는 토큰입니다.");
```